### ISSUE TITLE
Vimeo Video Embeds Not Showing On Page With 3.1.1

### ISSUE THREAD

**Issue by trynet** [elementor/elementor#13795]
Using the default video widget vimeo embeds are no longer visible on pages. Running WP 5.6 with no active plugins except free Elementor. (FYI  YouTube videos are still visible).  

Also using the current version of GeneratePress.

**Comment by Christopher-Hayes** [comment]
Same issue here. Vimeo embed is completely missing. Iframe doesn't appear to be present in the DOM, `#header-video` has `youtube_url` assigned to a Vimeo URL in `data-settings` for some reason.

![image](https://user-images.githubusercontent.com/6013871/106812008-25507480-663d-11eb-93b1-1c0562cced3e.png)

**Comment by Christopher-Hayes** [comment]
Found the problem I was experiencing. I'm not sure how it started, but all the video widgets using Vimeo had "YouTube" set as the provider. Pre-`3.1.0`, Elementor would still work with "YouTube" set as the provider, but Vimeo in the URL. `v3.1.0` brought that issue to light. I've no idea if I just overlooked that when I added the widget (weirdly multiple times too) or if the provider was added at some point, defaulted to YouTube, and this refactor brought that to light.

![image](https://user-images.githubusercontent.com/6013871/106831248-930c9880-665d-11eb-8301-8334c00a90d1.png)
